.env.example

```
@@ -0,0 +1,11 @@
+POSTGRES_DB=auth
+POSTGRES_USER=auth
+POSTGRES_PASSWORD=auth
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The POSTGRES_PASSWORD key should go before the POSTGRES_USER key

(UnorderedKey)